### ISSUE TITLE
Phantom crashes after 90 mins

### ISSUE THREAD

**Issue by matthewpalmer** [fraserh/waldi#9]
Good news is that this one isn't our fault. Here's the crash log

```
Saved.
phantom stderr: 2015-02-03 23:57:40.038 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.

phantom stderr: 2015-02-03 23:57:40.078 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.080 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.082 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.082 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.083 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.088 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.088 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.089 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.089 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.090 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.090 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.091 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.091 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.092 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.092 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.092 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.093 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.093 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.094 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.094 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.095 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.095 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.095 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.096 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.096 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.097 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.097 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.098 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.098 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.099 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.099 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.100 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.100 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.101 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.101 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.102 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.102 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.102 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.103 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.103 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.104 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.104 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.104 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.105 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.105 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.108 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.108 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.109 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.110 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.111 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.112 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.112 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.113 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.113 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.114 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.115 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.115 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.116 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.116 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.117 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.117 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.118 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.122 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.127 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.128 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.129 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.129 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.130 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.130 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.131 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.131 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.131 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.132 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.132 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.132 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.133 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.133 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.134 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.134 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.135 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.

phantom stderr: 2015-02-03 23:57:40.135 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.235 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.236 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.236 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.237 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.237 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.237 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.238 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.238 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.239 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.239 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.239 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.240 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.240 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.241 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.241 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.242 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.242 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.243 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.243 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.244 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.245 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.245 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.245 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.246 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.246 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.247 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.247 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.248 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.249 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.249 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.250 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.250 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.251 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.251 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.251 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.252 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.252 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.253 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.253 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.253 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.254 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.255 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.255 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.
2015-02-03 23:57:40.255 phantomjs[10640:446652] Critical failure: the LastResort font is unavailable.

phantom stdout: TypeError: 'undefined' is not a function (evaluating '$('.search-box-layout input.search-text').live('focus', function (event) {
    $('.search-box-layout').addClass('focus');
})')


phantom stdout:   http://www2.woolworthsonline.com.au/Scripts/Behaviour.js?v=5.23.6.0:1212

phantom stdout: ReferenceError: Can't find variable: browserFeatureDetection


phantom stdout:   http://www2.woolworthsonline.com.au/Shop/Browse/chilled?page=30:120

phantom stderr: PhantomJS has crashed. Please read the crash reporting guide at https://github.com/ariya/phantomjs/wiki/Crash-Reporting and file a bug report at https://github.com/ariya/phantomjs/issues/new with the crash dump file attached: /tmp/D87F1FD6-229E-4C19-BAE6-A30F3ED2D635.dmp

phantom stderr: ** Unknown exception behavior: 0


assert.js:92
  throw new assert.AssertionError({
        ^
AssertionError: signal killed phantomjs: SIGTRAP
    at Console.assert (console.js:102:23)
    at ChildProcess.<anonymous> (/Users/matthewpalmer/dev/waldi/crawler/node_modules/phantom/phantom.js:152:21)
    at ChildProcess.emit (events.js:98:17)
    at Process.ChildProcess._handle.onexit (child_process.js:810:12)
```

Now the file `/tmp/D87F1FD6-229E-4C19-BAE6-A30F3ED2D635.dmp` doesn't exist—it must have gotten cleaned up. I'll try to force the crash again tonight and hopefully I can crash the log file before it's gone.

**Comment by fraserh** [comment]
interesting

**Comment by matthewpalmer** [comment]
Fixed in ae414b9
